routes/cp.php

```diff
@@ -27,7 +27,16 @@
     Route::get('select-site/{handle}', 'SelectSiteController@select');
 
     Route::group(['namespace' => 'Structures'], function () {
-        Route::resource('navigation', 'NavigationController');
+        Route::resource('navs', 'NavigationController', ['names' => [
+            'index' => 'navigation.index',
+            'create' => 'navigation.create',
+            'store' => 'navigation.store',
+            'show' => 'navigation.show',
+            'edit' => 'navigation.edit',
+            'update' => 'navigation.update',
+            'destroy' => 'navigation.destroy',
+        ]]);
```

Comment:
There has to be a shorthand for this, right?

Reply:
Could not find, at least not for resource routes 😥